packages/nx-plugin/src/generators/library/typed-json-config/tsconfig.json.ts

```diff
@@ -7,8 +7,6 @@ export const packageTsConfig = (options: NormalizedSchema): TsConfig => ({
   compilerOptions: {
     baseUrl: 'src',
```

Comment:
Does tsconfig.build.json need to be updated as well?

Reply:
Yes, nicely spotted 👀